### PR TITLE
completes html-relative-links

### PR DESCRIPTION
**Pull request by aklepper20**
html-relative-links | klepper

image

### CODE REVIEW

File: html-relative-links/about-me/about-me.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li>Home Page</li>
+    <li>About Me</li>
+    <li>Projects</li>
+    <li>Contact Us</li>
```
Comment (codingcodymiller) [inline html-relative-links/about-me/about-me.html:16]: These still require links

File: html-relative-links/contact-us/contact-us.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li>Home Page</li>
+    <li>About Me</li>
+    <li>Projects</li>
+    <li>Contact Us</li>
```
Comment (codingcodymiller) [inline html-relative-links/contact-us/contact-us.html:16]: These still require links

File: html-relative-links/projects/projects.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li>Home Page</li>
+    <li>About Me</li>
+    <li>Projects</li>
+    <li>Contact Us</li>
```
Comment (codingcodymiller) [inline html-relative-links/projects/projects.html:16]: These still require links

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by thebearingedge** [comment]
@aklepper20 Please check out this branch, make a small change, commit, push, and open a new Pull Request so that @codingcodymiller can mark it as completed.

**Pull request merged**